persone.rb

```diff
@@ -1,14 +1,17 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id, :name, :age
+  attr_reader :id, :name, :age, :rentals
```

Comment:
- [ ] Hey there! Just a heads up 🚨: Make sure to include getters and setters for the @rentals attribute. 🔄 This will provide controlled access and modification to the rentals array in your class, ensuring a more secure and flexible implementation. Don't forget to implement those getters and setters! 👩‍💻📚

```ruby
attr_accessor :name, :age, :rentals
attr_reader :id
```